composer.json

```
@@ -19,18 +19,18 @@
         }
     ],
     "require": {
-        "php": "^7.2",
+        "php": "^7.2 || ^8.0",
         "ext-mongodb": "^1.1.5",
         "mongodb/mongodb": "^1.0",
         "symfony/framework-bundle": "^3.4 || ^4.3"
     },
     "require-dev": {
-        "matthiasnoback/symfony-dependency-injection-test": "^3",
+        "matthiasnoback/symfony-dependency-injection-test": "^3 || ^4",
         "symfony/web-profiler-bundle": "^3.4 || ^4.3",
         "symfony/console": "^3.4 || ^4.3",
         "phpunit/phpunit": "^7.0",
         "symfony/phpunit-bridge": "^4.2",
-        "facile-it/facile-coding-standard": "^0.4.0",
+        "facile-it/facile-coding-standard": "^0.4.0 || dev-master",
```

https://github.com/facile-it/facile-coding-standard/releases/tag/0.4.1
```suggestion
        "facile-it/facile-coding-standard": "^0.4.1",
```